versioned_docs/version-8.6/apis-tools/spring-zeebe-sdk/configuration.md

```diff
@@ -381,16 +381,19 @@ camunda:
       grpc-address: http://localhost:26500
 ```
 
+Make sure to add the scheme `http://` to the URL. Otherwise you will get a `java.lang.NullPointerException: target`.
```

Comment:
@camunda/tech-writers this could be wordsmithed a bit and turned into an admonition.